router/network.go

```diff
@@ -124,8 +124,16 @@ func (network *netHandle) SendPost(ctx context.Context, structData integrations.
 			case "FORM":
 				formValues := url.Values{}
 				for key, val := range bodyValue {
-					formValues.Set(key, fmt.Sprint(val)) // transformer ensures top level string values, still val.(string) would be restrictive
+					switch v := val.(type) {
+					case []string:
+						for _, listItem := range v {
+							formValues.Add(key, listItem)
+						}
+					default:
+						formValues.Add(key, fmt.Sprint(val))
+					}
 				}
+				headers["Content-Type"] = "application/x-www-form-urlencoded"
```

Comment:
Do we need this? Not mentioned in the PR description?

Reply:
normally transformer sets this but I thought, setting this from server is more apt as it is preparing the final payload